aspnetcore/razor-components/index.md

```diff
@@ -80,44 +52,85 @@ When this component is used elsewhere in the app, IntelliSense speeds developmen
+For apps that require third-party JavaScript libraries and browser APIs, Razor Components interoperate with JavaScript. Razor Components are capable of using any library or API that JavaScript is able to use. C# code can call into JavaScript code, and JavaScript code can call into C# code. For more information, see [JavaScript interop](xref:razor-components/javascript-interop).
 
 ## Optimization
 
-For client-side apps, payload size is critical. Blazor optimizes payload size to reduce download times. For example, unused parts of .NET assemblies are removed during the build process, HTTP responses are compressed, and the .NET runtime and assemblies are cached in the browser.
+Payload size is a critical performance factor for an app's useability. Blazor optimizes payload size to reduce download times:
+
+* Unused parts of .NET assemblies are removed during the build process.
+* HTTP responses are compressed.
+* The .NET runtime and assemblies are cached in the browser.
 
-Razor Components provides an even smaller payload size than Blazor by maintaining .NET assemblies, the app's assembly, and the runtime server-side. Razor Components apps only serve markup, script, and stylesheets to clients.
+Server-side Razor Components provides an even smaller payload size than Blazor by maintaining .NET assemblies, the app's assembly, and the runtime server-side. Razor Components apps only serve markup files and static assets to clients.
 
 ## Deployment
 
-Use Blazor to build a pure standalone client-side app or a full-stack ASP.NET Core app that contains both server and client apps:
+Build and deploy apps using either of two hosting models:
+
+* In a **Razor Components full-stack ASP.NET Core app**, host the app on any cloud or on-premise host supported by ASP.NET Core.
+* In a **standalone client-side Blazor app**, the app is compiled into a *dist* folder that only contains static files. Host the files on Azure App Service, GitHub Pages, IIS (configured as a static file server), Node.js, or any other server or service capable of serving static files. .NET isn't required on the server in production.
 
-* In a **standalone client-side app**, the Blazor app is compiled into a *dist* folder that only contains static files. The files can be hosted on Azure App Service, GitHub Pages, IIS (configured as a static file server), Node.js servers, and many other servers and services. .NET isn't required on the server in production.
-* In a **full-stack ASP.NET Core app**, code can be shared between server and client apps. The resulting ASP.NET Core Razor Components app, which serves the client-side UI and other server-side API endpoints, can be built and deployed to any cloud or on-premise host supported by ASP.NET Core.
+For more information, see <xref:host-and-deploy/razor-components/index>.
 
 ## Suggest a feature or file a bug report
```

Comment:
We don't generally ask for product feedback like this in the docs. We do that in the blog posts for preview releases.